migrations/20181013145007_add_rhapsody_table_schema.js

```diff
@@ -5,10 +5,10 @@ exports.up = function (knex, Promise) {
         if (!exists) {
             return knex.schema.createTable('rhapsody', function (table) {
                 table.bigincrements('id').primary();
-                table.string('title', 100);
+                table.string('title');
                 table.string('opening_verse');
                 table.text('body');
-                table.string('confession');
+                table.text('confession');

```

Comment:
we usually don't edit migrations , but rather create a new migration with the new changes.
we can allow this one to slip through this time only

Reply:
ok

Reply:
ok